src/sagas/publicChannels/publicChannels.selectors.ts

```diff
@@ -130,12 +130,102 @@ export const currentChannelDisplayableMessages = createSelector(
         id: message.id,
         type: message.type,
         message: message.message,
-        createdAt: formatMessageDisplayDate(message.createdAt),
+        createdAt: message.createdAt,
```

Comment:
Taking under consideration the fact that next selector is a duplicate, I guess you can revert the change in this line and keep 'displayableMessage' date in readable format (which was originally the goal of this selector)